### ISSUE TITLE
'toggle local view' when using `/`, that when running `bim.activate_drawing` that it goes to that view without throwing error. 

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#4311]
Would be nice, if in 'toggle local view' when using `/`, that when running `bim.activate_drawing` that it goes to that view without throwing following error. 

I might be able to tackle this one, if easy. 

```
Python: Traceback (most recent call last):
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\drawing\operator.py", line 1472, in invoke
    return self.execute(context)
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\bim\module\drawing\operator.py", line 1481, in execute
    core.activate_drawing_view(tool.Ifc, tool.Drawing, drawing=drawing)
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\core\drawing.py", line 429, in activate_drawing_view
    drawing_tool.activate_drawing(camera)
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\blenderbim\tool\drawing.py", line 1616, in activate_drawing
    bpy.ops.view3d.localview()
  File "C:\Program Files\Blender Foundation\Blender 3.6\3.6\scripts\modules\bpy\ops.py", line 113, in __call__
    ret = _op_call(self.idname_py(), None, kw)
RuntimeError: Operator bpy.ops.view3d.localview.poll() failed, context is incorrect

```

**Comment by Plae-Blue-Dot** [comment]
agreed this would be great

**Comment by theoryshaw** [comment]
noticed this stopped working with 4.1

**Comment by BIMvoice** [comment]
This is already fixed on current v0.8.0. The traceback came from an unguarded bpy.ops.view3d.localview() call in activate_drawing that failed its poll when you were already in local view (pressed /). The drawing activation flow was since refactored: it no longer calls the local view operator at all, and where it does toggle local view (tool.Blender.activate_camera) it is guarded on the current state and run inside a proper context override, so the poll cannot fail. Closable as resolved by the activate_drawing refactor.

**Comment by BIMvoice** [comment]
Closing as resolved by the activate_drawing refactor (see comment above). Feel free to reopen if this still reproduces on current v0.8.0.
